test/util/e2e.go

```diff
@@ -24,6 +24,11 @@ import (
 	visibilityv1alpha1 "sigs.k8s.io/kueue/client-go/clientset/versioned/typed/visibility/v1alpha1"
 )
 
+const (
+	// Sleep image to use for testing.
+	E2eTestSleepImage = "gcr.io/k8s-staging-perf-tests/sleep@sha256:8d91ddf9f145b66475efda1a1b52269be542292891b5de2a7fad944052bab6ea"
```

Comment:
Would it work to include both the tag (version) and the sha?
Like this: `gcr.io/k8s-staging-perf-tests/sleep:v0.1.0@sha256:8d91ddf9f145b66475efda1a1b52269be542292891b5de2a7fad944052bab6ea`
I think this would improve human readability.